builder/azure/arm/builder.go

```diff
@@ -64,6 +64,12 @@ func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (pack
 		return nil, err
 	}
 
+	//When running Packer on an Azure instance FillParameters will update SubscriptionID from the instance
```

Comment:
It would help to mention this is particularly the case when using managed identity.
```suggestion
	//When running Packer on an Azure instance using Managed Identity, FillParameters will update SubscriptionID from the instance
```